polar-core/src/bindings.rs

```diff
@@ -425,52 +427,91 @@ impl BindingManager {
     }
 }
 
+fn combine(left: &Symbol, l: Operation, right: &Symbol, r: Operation) -> Operation {
```

Comment:
needs a better name

Reply:
What's this method doing? A comment would be super helpful.

Reply:
👍 that sounds fine. and a comment

Reply:
replaced this entire function with `term!(op!(And, term!(left), term!(right)))`